packages/sources/finage/src/adapter.ts

```diff
@@ -51,7 +51,9 @@ export const makeWSHandler = (config?: Config): MakeWSHandler<Message | any> =>
       }
     }
     const isEtf = (input: AdapterRequest): boolean =>
-      !!input.data.endpoint && endpoints.ukEtf.supportedEndpoints.includes(input?.data?.endpoint)
+      !!input.data.endpoint &&
+      (endpoints.etf.supportedEndpoints.includes(input?.data?.endpoint) ||
```

Comment:
[low prio] probably unnecessary, but if you wanted to maintain the single check formatting (instead of OR), you could pre-compute a
`const etfEndpoint = [...etf.supportedEndpoints, ...ukEtf.supportedEndpoints]`
and then use it instead of the 2 separate checks.

Reply:
Good suggestion, added!